modules/ROOT/pages/api-led-develop.adoc

```diff
@@ -3,108 +3,107 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You've created an API specification, now use it to create an API.
+Now that you've created an API specification, use it to create a Mule app.
 
 In Anypoint Platform, developers package an API in an application that is deployed to an instance of xref:mule-runtime::index.adoc[Mule runtime engine (Mule)].
-Mule is a lightweight integration engine that runs Mule apps. Instances of Mule are embedded in Studio, and in the environments where you will deploy Mule apps and APIs.
+Mule is a lightweight integration engine that runs Mule apps. Instances of Mule are embedded in Studio, and in the environments where you deploy Mule apps and APIs.
 These environments are called targets. One target is the Mule embedded in Studio, which you use for
 development and testing. Another target, xref:runtime-manager::cloudhub.adoc[CloudHub], is managed by MuleSoft and is cloud based.
 
-This tutorial shows you how to deploy first to Mule in Studio. When the API is complete, you'll deploy it to Mule in CloudHub.
+You are going to deploy first to Mule in Studio, then, when the app is complete, you'll deploy it to Mule in CloudHub.
 
 image:empty-studio.png[Studio without any projects]
 
-. The Package Explorer helps you navigate the files created for each project.
-. The canvas is where you drag and drop elements of your project and connect them, creating a flow.
+. Package Explorer helps you navigate the files created for each project.
```

Comment:
Is this the case where you calloutlist didn't work?
https://confluence.internal.salesforce.com/display/MTDT/Formatting+with+Asciidoc#FormattingwithAsciidoc-AddCalloutstoCodeBlocks

Reply:
No, I didn't make any changes here.